libcoap-2.map

```diff
@@ -137,7 +141,6 @@ global:
   coap_print_addr;
   coap_print_link;
   coap_print_wellknown;
-  coap_read;
```

Comment:
Maybe it is a bit too early to remove the deprecated functions `coap_read`, `coap_run_once`, and `coap_write`?

Reply:
These are now inline functions calling the appropriate new function defined in include/coap2/net.h.  Hence no longer defined in libcoap-2.map.

Reply:
Of course. Sorry, I missed that.